README.md

```diff
@@ -47,6 +47,7 @@ Options:
   -acl, --access-control-level  Sets the access control level for uploaded files
                                                                                                   [string] [default: "undefined"]
   -c, --config         The AWS config json path to load S3 credentials with loadFromPath.                       [string]
+  -o, --overwrite      Overwrite remote files with the same name, or skip them.                [boolean] [default: true]
```

Comment:
Flags that are true by default always feel a big awkward, since you always only use the negative version of it, like `--no-overwrite` in this case.

I think it's worth to explicitly add this to the description, and also as an example more blow (line 60).

Reply:
Agreed that having a `overwrite: true` as a default makes this a bit awkward, but I wanted to avoid double negations and keep backward compatibility. I have changed the description to:

> Overwrite remote files with the same name (default behavior), or skip them with --no-overwrite.

And added example.